easybuild/easyconfigs/r/RSeQC/RSeQC-5.0.4-foss-2024a.eb

```
@@ -0,0 +1,60 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Adam Huffman
+# The Francis Crick Institute
+# Update: Petr Král (INUITS)
+# Update: J. Saßmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'RSeQC'
+version = '5.0.4'
+
+homepage = 'http://rseqc.sourceforge.net/'
+description = """RSeQC provides a number of useful modules that can
+ comprehensively evaluate high throughput sequence data especially RNA-seq
+ data. Some basic modules quickly inspect sequence quality, nucleotide
+ composition bias, PCR bias and GC bias, while RNA-seq specific modules
+ evaluate sequencing saturation, mapped reads distribution, coverage
+ uniformity, strand specificity, transcript level RNA integrity etc."""
+
+toolchain = {'name': 'foss', 'version': '2024a'}
+
+builddependencies = [
+    ('hatchling', '1.24.2'),
+]
+
+dependencies = [
+    ('Python', '3.12.3'),
+    ('Pysam', '0.22.1'),
+    ('bx-python', '0.13.0'),
+    ('pyBigWig', '0.3.24'),
+    ('matplotlib', '3.9.2'),
+]
+
+exts_list = [
+    ('logomaker', '0.8.7', {
+        'checksums': ['63783ce6e24449d6f1f01ce29c4ae1b91f6e54bc198e5da40ad18fcd0efc3302'],
+    }),
+    (name, version, {
+        'sources': [SOURCELOWER_TAR_GZ],  # this is different from previous versions, maybe mistake?
```

it's not a mistake, the name of the tarball was changed, so you can remove the comment
https://files.pythonhosted.org/packages/c6/52/b1e6a87d516849971376966b4475c878303665a3d600a35600209cd6c9af/RSeQC-5.0.1.tar.gz
https://files.pythonhosted.org/packages/f8/04/f32be86c3e14fa77b0dae3a9cdea9e5795642960ade79e3d463664101a4d/rseqc-5.0.4.tar.gz

```suggestion
        'sources': [SOURCELOWER_TAR_GZ],
```